app/views/layouts/_top_header.html.haml

```diff
@@ -0,0 +1,46 @@
+.service-header
+  .div-nav
+    %nav.navbar.navbar-static-top.navbar-expand-lg.navbar-light
+      %div
+        = image_tag('coffee.jpg', class: "d-inline-block align-top logo-img")
+        = link_to home_path, id: 'logo-text', class: 'navbar-brand texty' do
+          EUREKA
+          %br/
+          CAFFEE
+      %button.navbar-toggler{ 'aria-controls' => 'navbar-nav', 'aria-expanded' => 'false', 'aria-label' => 'Toggle navigation', 'data-target' => '#navbar-nav', 'data-toggle' => 'collapse', :type => 'button' }
+        %span.navbar-toggler-icon
+      #navbar-nav.collapse.navbar-collapse.bg-custom
+        %ul#menu-top.navbar-nav.ml-auto
+          %li.nav-item.active
+            = link_to home_path, class: 'nav-link texty' do
+              = content_tag(:span, '·', class: 'middot')
+              HOME
+          - if logged_in?
+            %li.nav-item.dropdown
+              %a.dropdown-toggle.texty.nav-link{ 'data-toggle' => 'dropdown', :href => '#' }
+                Account
+              %ul.dropdown-menu.texty
+                %li= link_to 'Profile', current_user
+                %li= link_to 'Settings', edit_user_path(current_user)
+                %li
+                  = link_to 'Log out', logout_path, method: :delete
+            - if current_user.admin? && !current_user?(@user)
+              %li.nav-item.active
+                = link_to home_path, class: 'nav-link texty' do
+                  = content_tag(:span, '·', class: 'middot')
+                  Dashboard
+          - else
+            %li.nav-item
+              = link_to 'Log in', login_path, :class =>'nav-link texty'
+            %li.nav-item
+              = link_to 'Sign up', new_user_path, :class =>'nav-link texty'
+  .jumbotron.text-center
+    %div
+      %h1.title.texty Eureka Caffee
+      = image_tag 'lines.png', style: 'filter:invert(100%)', width: '50px'
+      = image_tag 'lines.png', style: 'filter:invert(100%)', width: '50px'
+      %i.fas.fa-utensils
+      = image_tag 'lines.png', style: 'filter:invert(100%)', width: '50px'
```

Comment:
NEVER USE INLINE STYLING AGAIN. I told you this too many times. Please pay attention to my comments. I don't want to repeat myself because I feel like I'm doing all this work for nothing. 

This is bad because all this code is duplicated, you can't interact with it, you can't reuse it, you can't cascade it, you can't make it responsive, you can't use variables. Should I go on with reasons why you should never use inline styling?

From now on I'll review only commits that do NOT have inline styling

Ping me when I can continue reviewing this pull request

Reply:
Kind of solved the in-line styling & repetition. ✓
Error persisted from previous PR due to being in a rush, so my bad.
But, as a funny side-fact, it's the only in-line-styling in the whole project.